library/src/mixins/property/method_file_split/using_method_regex_fy.py

```diff
@@ -42,4 +39,13 @@ def _method_file_split(self) -> List[str]:
             len(method_file_split)
         ) == 7, f"Method file split length {len(method_file_split)} is invalid."
 
-        return method_file_split
+        method_file_split_model = MethodFileSplitModel(
+            user_imports=method_file_split[0],
+            method_name=method_file_split[1],
+            implementation_name=method_file_split[5],
+            arguments=method_file_split[3],
+            return_type=method_file_split[4],
+            mixin_split=method_file_split[6].split("\n"),
```

Comment:
Let's keep the original string so if anyone needs it for a different purposes, like parsing, they can use it. This way no one has the original string.

Add split in the place where it is consumed.